Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -211,6 +233,27 @@ void SetupIDs()
             }
         }
 
+        internal void FindAvailableSceneId(bool overrideCheck = false)
+        {
+            // Only find a new scene ID if one is not set or another sceneId has our
+            // sceneId because we were duplicated in editor
+            if (this.sceneId == 0 || overrideCheck)
+            {
+                bool valueFound = false;
+                uint counter = 1;
+                while (!valueFound)
+                {
+                    if (!NetworkIdentityManager.Instance.DoesSceneIdExists(counter))
+                    {
+                        Debug.LogFormat("[NetworkIdentity] Found SceneId {0} for {1}", counter, this.gameObject.GetHierarchyPath());
```

Comment:
what is GetHierarchyPath? doesn't seem to be a GameObject function

Reply:
Sorry, that hasn't been added yet. I'll do that today.  It prints out the path of the game object in the Hierarchy like `Environment\City\Buildings\Build01\Window`

Reply:
This is checked in now.